Libraries/LibWeb/WebIDL/Promise.cpp

```diff
@@ -209,8 +209,8 @@ GC_DEFINE_ALLOCATOR(WaitForAllResults);
 // https://webidl.spec.whatwg.org/#wait-for-all
 void wait_for_all(JS::Realm& realm, Vector<GC::Ref<Promise>> const& promises, Function<void(Vector<JS::Value> const&)> success_steps, Function<void(JS::Value)> failure_steps)
 {
-    // FIXME: Fix spec typo, fullfilled --> fulfilled
-    // 1. Let fullfilledCount be 0.
+    // FIXME: Fix spec typo, fulfilled --> fulfilled
```

Comment:
OK it looks like these typo fixes were entirely automated. :sweat_smile: If you're going to do that, please check your work. These ones in `wait_for_all()` are a bit silly.

Reply:
Sorry for that.
Every change is automatic fix but I've missed this during review.